RemoteSettings.Dockerfile

```
@@ -27,9 +27,22 @@ COPY ./kinto-remote-settings ./kinto-remote-settings
 COPY version.json .
 RUN pip install ./kinto-remote-settings
 
+# We build the Kinto Admin assets at the specific
+# version specified in `kinto-admin/VERSION`.
+FROM node:20.11.0-slim as build-admin
+COPY /bin/update_and_install_system_packages.sh /opt
+RUN /opt/update_and_install_system_packages.sh \
+    ca-certificates\
+    curl
+COPY bin/build-kinto-admin.sh /opt/
+COPY kinto-admin/ /opt/kinto-admin/
+RUN cd /opt && ./build-kinto-admin.sh
```

- I think if we use the non-slim `node` base container, we can avoid downloading `curl`. Since we're copying the built assets from that container, it seems fine to use the non-slim version
- [Hadolint](https://github.com/hadolint/hadolint) (which I run locally) poked me about setting a `WORKDIR`. Doing so seemed like a nice change?

```suggestion
FROM node:20.11.0 as build-admin
WORKDIR /opt
COPY bin/build-kinto-admin.sh .
COPY kinto-admin/ kinto-admin/
RUN ./build-kinto-admin.sh
```